Opensourcecommunity/Communication-and-contact.md

```diff
@@ -37,4 +37,4 @@ If you have a question, please create an Github issue in the [documentation repo
 
  
 ### Report security issues
-Due to the sensible nature of security issues e.g. zero days, we prefer a responsible disclosure. Security issues can be reported on the [opensmartgridplatform.org website](https://opensmartgridplatform.org/report-security-issue/).
+Due to the sensible nature of security issues e.g. zero days, we prefer a responsible disclosure. Security issues can be reported on the one of the maintainers personally.
```

Comment:
can be reported on the one of the maintainers personally -> can be reported at one of the maintainers personally